src/validation/ValidationApi.js

```diff
@@ -12,14 +12,20 @@ class ValidationApi {
       valid: true
     };
 
-    const res = await this.http.post(this.validationApiUrl, {
-      body: json,
-      json: true
-    });
+    try {
+      const res = await this.http.post(this.validationApiUrl, {
+        body: json,
+        json: true
+      });
 
-    if (!isEmpty(res)) {
+      // TODO: remove after ONSdigital/eq-schema-validator/pull/42 is merged
```

Comment:
If you merge this then you can remove this code before merge.

Reply:
We need to be forwards and backwards compatible briefly. If we remove this now, then we won't be handling errors properly in the period between this merge/deploy and validator merge/deploy

Reply:
That window would be about 1 minute between merges? as the other PR is ready to go?
You will need to raise another PR now